spec/octopus/serverspec/type/octopus_deploy_project_group_spec.rb

```diff
@@ -72,46 +95,25 @@
           to_return(status: 200, body: ex_spaces_all, headers: {})
 
 
-      pg = OctopusDeployProjectGroup.new("https://octopus.example.com", "API-1234567890", "Octopus Projects", "Octopus")
+      pg = OctopusDeployProjectGroup.new("https://octopus.example.com", "API-1234567890", "Octopus Projects").in_space('Octopus')
```

Comment:
Worth having a test here to make sure its a nice error if server doesn't support spaces?

Reply:
Added a new `raise` if we can't resolve the Space Id, and moved the raise if we're not provided a `project_group_name` because that was in a stupid place